site/content/docs/5.3/examples/e-shop/index.html

```diff
@@ -0,0 +1,1567 @@
+      <div class="col-12 pt-3">
+        <div class="row gx-0 bg-dark">
+          <div class="col-12 col-lg-7 order-lg-1" aria-hidden="true">
+            <div class="ratio ratio-16x9">{{< placeholder width="100%" height="162" text="icon" >}}</div>
+          </div>
+          <div class="col-12 col-lg-5">
+            <div class="h-100 p-2 p-lg-4">
+              <div class="pt-1 pt-lg-0 pe-3 pe-lg-4 pb-1 ps-2">
+                <h3 class="h1 mb-2 text-primary">The best for my budget</h3>
+                <p class="h2 pt-1 pt-lg-2 mb-2">iPhone at a reduced price refurbished, with 1 year guarantly</p>
+                <p class="h5 pt-1 pt-lg-2 mb-3 text-primary">From 850 000 FCFA</p>
+                <a href="#" class="btn btn-secondary btn-inverse mt-1 mt-lg-2 mb-2">Find yours</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-12 pt-3 pt-lg-5">
+        <div class="row gx-0 bg-body">
+          <div class="col-12 col-lg-7 order-lg-1" aria-hidden="true">
+            <div class="ratio ratio-16x9">{{< placeholder width="100%" height="162" text="icon" >}}</div>
+          </div>
+          <div class="col-12 col-lg-5">
+            <div class="h-100 pt-2 pt-lg-0">
+              <div class="pt-1 pt-lg-0 pe-lg-5">
+                <h3 class="mb-2">Why buy your phone from Orange?</h3>
+                <p class="lead mb-0 mt-lg-1 line-height-custom">Come quickly to our Orange Stores to discover the wide range of phones and smartphones that Orange offers you. Buying a new mobile phone from Orange is guaranteed to benefit from all the Orange services, its after-sales service and a welcome bonus. Our advisers are waiting to help you choose the mobile that suits you ans demonstrate it to you.</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div class="col-12 pt-2 pt-lg-4">
+        <div class="row gx-0 bg-body">
+          <div class="col-12 col-lg-7" aria-hidden="true">
+            <div class="ratio ratio-16x9">{{< placeholder width="100%" height="162" text="icon" >}}</div>
+          </div>
+          <div class="col-12 col-lg-5">
+            <div class="h-100 pt-2 pt-lg-0">
+              <div class="pt-1 pt-lg-0 ps-lg-5">
+                <h3 class="mb-2">How to choose your smartphone?</h3>
+                <p class="lead mb-0 mt-lg-1 line-height-custom">Discover our wide choice of Android, Samsung or Apple iPhone smartphones to take advantages of the quality and speed of Orange's 4G network. Buying a smartphone from Orange is guaranteed to benefit from the quality of the latest generation cameras, and a large storage capacity. The long-lasting batteries will allow you to take advantage of all the features of your mobile to listen to music, watch videos and take advantage of social networks. Finally, the smartphone will be the ideal companion for paying, transferring or withdrawing money with Orange Money.</p>
```

Comment:
'Android, Samsung or Apple' => Samsung is not using Android anymore?

Reply:
TBD with designers